src/python/aqueduct_executor/operators/connectors/data/sqlite.py

```diff
@@ -1,12 +1,35 @@
-from aqueduct_executor.operators.connectors.data import config, relational
+import pandas as pd
+from aqueduct_executor.operators.connectors.data import config, load, relational
+from aqueduct_executor.operators.utils.enums import ArtifactType
 from sqlalchemy import create_engine, engine
 
+SQLITE_MAX_VARIABLE_NUMBER = 32766
+
 
 class SqliteConnector(relational.RelationalConnector):
     def __init__(self, config: config.SqliteConfig):
         conn_engine = _create_engine(config)
         super().__init__(conn_engine)
 
+    def load(
+        self, params: load.RelationalParams, df: pd.DataFrame, artifact_type: ArtifactType
+    ) -> None:
+        if artifact_type != ArtifactType.TABLE:
+            raise Exception("The data being loaded must be of type table, found %s" % artifact_type)
+        # NOTE (saurav): df._to_sql has known performance issues. Using `method="multi"` helps incrementally,
+        # since pandas will pass multiple rows in a single INSERT. If this still remains an issue, we can pass in a
+        # callable function for `method` that does bulk loading.
+        # See: https://pandas.pydata.org/docs/user_guide/io.html#io-sql-method
+        df.to_sql(
+            params.table,
+            con=self.engine,
+            if_exists=params.update_mode.value,
+            index=False,
+            method="multi",
+            # We need to specify chunksize due to sqlite3's variable number limit.
+            chunksize=int(SQLITE_MAX_VARIABLE_NUMBER / len(df.columns)),
```

Comment:
For my understanding (and maybe we could also add a comment about this calculation), it's one variable per column, so this restricts the number of rows such that the total number of variables is under the limit?

Reply:
yup, will add a comment.